contrib/juliac-trim-base.jl

```diff
@@ -78,38 +78,12 @@ end
         end
     end
     show_type_name(io::IO, tn::Core.TypeName) = print(io, tn.name)
-
-    mapreduce(f::F, op::F2, A::AbstractArrayOrBroadcasted; dims=:, init=_InitialValue()) where {F, F2} =
```

Comment:
Are these static parameters no longer needed to make code trimmable? If not, great. If they are, maybe we should put them in Base. I doubt we gain much by trying not to specialize on these.